tests/framework/installer/ceph_manifests.go

```diff
@@ -2065,6 +2071,37 @@ parameters:
     clusterNamespace: ` + namespace
 }
 
+func (m *CephManifestsMaster) GetBlockSnapshotClassDef(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
```

Comment:
How about removing the `Def` suffix on any of these new methods? We could remove it from the existing methods too if you want for consistency.
```suggestion
func (m *CephManifestsMaster) GetBlockSnapshotClass(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
```